docs/governance/community-guidelines/releases/README.md

```diff
@@ -0,0 +1,81 @@
+# Releases
+
+This document is a [community guideline].
+
+## Purpose
+
+The document outlines the guidelines of managing releases within the Common Cloud Controls (CCC) project.
+
+## Ownership
+
+The [Communications WG] has overall ownership of releases. They are responsible for setting the roadmap and communicating project progress to the wider community. All other [WG]'s should contribute to determining the content of a release.
```

Comment:
```suggestion
The [Communications WG] has overall ownership of releases. They are responsible for setting the roadmap and communicating project progress to the wider community. All other [WG]'s should contribute to determining the content of a release.
```

Curious about the meaning behind this: "The Communications WG has overall ownership of releases."

Reply:
I think the initial thinking behind this was the comms WG would be the folk speaking to the users and stakeholders to get requests, maintain the roadmap and then be commuinciating back to the users/stakeholders so they would own the release in this regard.